types/react-swipeable/index.d.ts

```diff
@@ -1,31 +1,24 @@
-// Type definitions for react-swipeable 3.3.1
+// Type definitions for react-swipeable 4.0
 // Project: https://www.npmjs.com/package/react-swipeable
-// Definitions by: Giedrius Grabauskas <https://github.com/GiedriusGrabauskas>
+// Definitions by: Giedrius Grabauskas <https://github.com/GiedriusGrabauskas>,
```

Comment:
Comma not needed.
https://github.com/DefinitelyTyped/DefinitelyTyped#edit-an-existing-package